### PR TITLE
Add tag with env_code

### PR DESCRIPTION
**Pull request by mohanedmoh**

### CODE REVIEW

File: Terraform/AWS/task3/VPCinAWS.tf [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+
+resource "aws_route_table_association" "private_association1" {
+  subnet_id      = aws_subnet.private1.id
+  route_table_id = aws_route_table.private1.id
+}
+resource "aws_route_table_association" "private_association2" {
+  subnet_id      = aws_subnet.private2.id
+  route_table_id = aws_route_table.private2.id
+}
```
Comment (geoffrey1330) [inline Terraform/AWS/task3/VPCinAWS.tf]: Add newline at the end of the file.

File: Terraform/AWS/task3/terraform.tfvars [hunk 2]
```diff
@@ -0,0 +1 @@
+env_code = "Mentorship"
```
Comment (geoffrey1330) [inline Terraform/AWS/task3/terraform.tfvars]: Add newline at the end of the file.

File: Terraform/AWS/task3/variables.tf [hunk 3]
```diff
@@ -0,0 +1,3 @@
+variable env_code{
+    type = string
+}
```
Comment (geoffrey1330) [inline Terraform/AWS/task3/variables.tf]: Add newline at the end of the file.

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: Terraform/AWS/task3/variables.tf (hunk 3 above)
Comment (mohanedmoh) [reply Terraform/AWS/task3/variables.tf]: Hi @geoffrey1330 can you explain why should I add a new line at the end of each file

Comment (geoffrey1330) [reply Terraform/AWS/task3/variables.tf]: It is a requirement by git for a good code style.

Comment (mohanedmoh) [reply Terraform/AWS/task3/variables.tf]: okay thank you

**Review by geoffrey1330 — APPROVED** [review]

**Pull request merged**